Sources/gRPC/Channel.swift

```diff
@@ -77,3 +80,5 @@ public class Channel {
     return Call(underlyingCall: underlyingCall, owned: true, completionQueue: completionQueue)
   }
 }
+
+private let defaultUserAgent = "\(grpc_g_stands_for()):\(grpc_version_string())"
```

Comment:
I would make this more prominent. Maybe move it to the top of the file or into the `gRPC` class. Also, would you mind using the `gRPC.gStandsFor` and `gRPC.version` variables instead? (I will file a separate PR to make them non-optional again.)